sinks/datadog/datadog.go

```diff
@@ -31,6 +34,17 @@ const datadogSpanType = "web"
 // we can flush per flush-interval
 const datadogSpanBufferSize = 1 << 14
```

Comment:
Safe to delete?

Reply:
This was left unused because I forgot to add the default span buffer size behavior in the original changes. Thanks for catching this! I've added the functionality back.

Reply:
Nice. Could we add a unit test to prevent this kind of regression? I don't feel strongly and the default isn't that critical so I'll let you decide.